bin/physlr-make

```diff
@@ -909,9 +918,12 @@ endif
 %.mol2-bkcs.tsv: %.tsv %.junctions.tsv
 	$(time) $(python) $(bin)/physlr molecules -V$V -t$t --separation-strategy=bc+k3+cos+sqcos $^ >$@
 
+prune_branches=10
```

Comment:
So we needed to add in these checks to make the integration test work? I think we should have these at the top of the file with the other parameter declarations

Reply:
I dont think the users should touch these variables just as they dont touch `minimizer_multiplicity`

Reply:
Well then maybe put them by the test code instead? Just seems strange to have them here by the pruning code if we wouldn't actually touch them in any other situation other than the test

Reply:
Those values are default values of `prune_branches`. Previously they were hardcoded

Reply:
Yes I know -- We kept them hardcoded because we didn't want to change them in the normal physlr case. If we do _only_ want to change them in the test situation, then I think it makes more sense to put them by that relevant code.